### DISCUSSION TITLE
article list page

### DISCUSSION THREAD

**Discussion by luweiqianyi** [vuepress/core#1417] [Q&A]
### Clear and concise description of the problem

Is it possible to develop a component that displays all articles instead of specifying it by using the sidebar?

### Suggested solution

Is it possible to develop a component that displays all articles instead of specifying it by using the sidebar?

**Comment by Mister-Hope** [comment]
You'd better describe what you want carefully, for now, vuepress has:

- components that can generate catalog for you like vuepress-plugin-auto-catalog
- theme that can automatically generate sidebar from file structure like vuepress-theme-hope

Find more at https://marketplace.vuejs.press/

This is not a bug, and we are not "your coder", using discussion to clearly describe what you need should be the correct behavior.
